scripts/gen_jvt.py

```diff
@@ -37,7 +37,6 @@
 BASE_URL = "https://www.itu.int/"
 H264_URL = BASE_URL + "wftp3/av-arch/jvt-site/draft_conformance/"
 BITSTREAM_EXTS = (
-    ".bit",
```

Comment:
Don't we have any .bit files in H264 test suites?

Reply:
No... I forgot remove it (in your comment) in the last PR:

https://github.com/fluendo/fluster/pull/177